utils/wasi-nn/install-openvino.sh

```diff
@@ -3,17 +3,17 @@
 # SPDX-FileCopyrightText: 2019-2022 Second State INC
 
 if [[ ! -v "${OPENVINO_VERSION}" ]]; then
```

Comment:
Since these two variables are useless now, we should remove them directly.

Reply:
done